pkg/kubectl/cmd/certificates.go

```diff
@@ -175,6 +175,10 @@ func (options *CertificateOptions) modifyCertificateCondition(f cmdutil.Factory,
 		Flatten().
 		Latest().
 		Do()
+	err = r.Err()
```

Comment:
i see a mixed pattern on how we chose to handle this, but isnt the visit function effectively ending up doing the same thing internally?

Reply:
@derekwaynecarr yeah, in the visit function there also has err check. But like other command, if the `Result` has err, we should end up immediately.